### PR TITLE
Add calls to mock item and add get lambda for mock

### PR DESCRIPTION
**Pull request by viggo-mathem**
To help verify that a lambda called the specified http mock.

### CODE REVIEW

File: backend/src/GetMock.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+const DynamoDB = require('aws-sdk/clients/dynamodb');
+const dynamoDb = new DynamoDB.DocumentClient();
+
+exports.handler = async function (event, context) {
+	console.log(event.path);
```
Comment (jaknor) [inline backend/src/GetMock.js]: Do we need these logs? If so can we fix the indentation?

**Review by jaknor — APPROVED** [review]
Looks good, just a question about some logging

File: backend/src/GetMock.js (hunk 1 above)
Comment (viggo-mathem) [reply backend/src/GetMock.js]: Removed

File: backend/template.yaml [hunk 2]
```diff
@@ -42,6 +42,22 @@ Resources:
...
+      Handler: GetMock.handler
+      Environment:
+        Variables:
+          TableName: !Ref MockConfig
+      Events:
+        MockRoute:
+          Type: Api
+          Properties:
+            Method: get
+            Path: '/{proxy+}'
```
Comment (daweij) [inline backend/template.yaml]: Should we prefix this with `/get/`? I'm just comparing to the other endpoints.

**Review by daweij — APPROVED** [review]

File: backend/template.yaml (hunk 2 above)
Comment (viggo-mathem) [reply backend/template.yaml]: Done

**Review by ljacobsson — APPROVED** [review]
:+1:

**Pull request merged**